conda/recipes/libcudf/meta.yaml

```diff
@@ -31,12 +31,10 @@ requirements:
     - cudatoolkit {{ cuda_version }}.*
     - arrow-cpp 1.0.1.*
     - arrow-cpp-proc * cuda
-    - flatbuffers
```

Comment:
Do we no longer need this?

Reply:
This was needed to build Arrow from source which we no longer do in the context of a conda build.